app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
+  layout 'mailer'
+end
```

Comment:
I am guessing these are also created by rails scaffold generator - you can also delete this unless you are planning to use this in forgot password of your login page (it did not go so well for me 😫 )